labman/db/tests/test_process.py

```diff
@@ -187,8 +176,7 @@ def test_notes(self):
 class TestReagentCreationProcess(LabmanTestCase):
     def test_attributes(self):
         tester = ReagentCreationProcess(5)
-        self.assertEqual(tester.date,
-                         _help_make_datetime('2017-10-23 09:10:25-0700'))
+        self.assertEqual(str(tester.date), '2017-10-23 09:10:25')
```

Comment:
If we want to move to testing the dates as *strings*, then I think all the changed code instances like this one, here and below, are impacted by above comments re removing seconds from expected string values of the dates and re possibly keeping `_help_format_datetime`. 

Alternatively, we can keep testing them as date objects, and just change `_help_make_datetime` to strip and/or ignore the timezone information when doing the conversion to an object.

Reply:
... k ...